src/components/WithBackgroundImage/index.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
+import useThemeContext from '@theme/hooks/useThemeContext';
+
+const WithBackgroundImage = ({
+  children,
+  imageDark,
+  imageLight, 
+  tag: Tag, 
+  ...props
+}) => {
+  const {isDarkTheme} = useThemeContext();
+
+  const backgroundImage = isDarkTheme && imageDark 
+    ? imageDark 
+    : imageLight;
+
+  return (
+    <Tag
+      style={backgroundImage 
```

Comment:
nit: Make `backgroundImage` a const, pulling it out of the jsx

Reply:
Good catch... I really don't like having logic in JSX